src/test/kotlin/com/igorwojda/integer/generateallpairs/challenge.kt

```diff
@@ -4,7 +4,13 @@ import org.amshove.kluent.shouldBeEqualTo
 import org.junit.jupiter.api.Test
 
 private fun getAllPairs(n: Int): List<Pair<Int, Int>> {
-    TODO("not implemented")
+    val list = arrayListOf<Pair<Int, Int>>()
+    for (firstElem in 0..n){
+        for ( secondElem in 0..n){
+            list.add(Pair(firstElem, secondElem))
```

Comment:
Can you think of an infix function (https://kotlinlang.org/docs/reference/functions.html#infix-notation) that can make your code a lil bit more readable?

Reply:
I have now found out about the existence of "to". I am now using it for creating the pairs: 
`list.add(firstElem to secondElem)`